lib/dartdoc.dart

```diff
@@ -19,19 +19,20 @@ import 'package:html/parser.dart' show parse;
 import 'package:path/path.dart' as pathLib;
 
 import 'package:tuple/tuple.dart';
-import 'src/config.dart';
-import 'src/generator.dart';
-import 'src/html/html_generator.dart';
-import 'src/logging.dart';
-import 'src/model.dart';
-import 'src/package_meta.dart';
-import 'src/warnings.dart';
-
-export 'src/config.dart';
-export 'src/element_type.dart';
-export 'src/generator.dart';
-export 'src/model.dart';
-export 'src/package_meta.dart';
+import 'package:dartdoc/src/config.dart';
```

Comment:
Consider re-ordering alphabetically?

FWIW IDEA makes this a snap w/ the "Optimize Imports..." action.

![image](https://user-images.githubusercontent.com/67586/38962394-efebc326-4321-11e8-9312-5951fd5c19f2.png)

Reply:
done